examples/ZemiSmart/Makefile

```diff
@@ -0,0 +1,18 @@
+PROGRAM = light
+
+EXTRA_COMPONENTS = \
+	extras/http-parser \
+	$(abspath ../../components/wolfssl) \
+	$(abspath ../../components/cJSON) \
+	$(abspath ../../components/homekit)
+
+LIBS ?= hal m
+FLASH_SIZE = 8
```

Comment:
Could you change variable initialization to "?=":
```
FLASH_SIZE ?= 8
HOMEKIT_SPI_FLASH_BASE_ADDR ?= 0x7A000
```
This allows overriding them at build time. I just pushed changes to all other examples to support that, so it would be nice if this would be consistent with them.